pages/translations/en/LC_MESSAGES/messages.po

```diff
@@ -1147,7 +1150,1930 @@ msgstr ""
+msgstr ""
+
+msgid "Don’t break the web."
+msgstr ""
+
+msgid "Ensure that if AMP has outages or problems it doesn’t hurt the rest of the web. That means if the Google AMP Cache, the URL API or the library fails it should be possible for websites and consumption apps to gracefully degrade. If something works with an AMP cache it should also work without a cache."
+msgstr ""
+
+msgid "Solve problems on the right layer."
+msgstr ""
+
+msgid "E.g. don’t integrate things on the client side, just because that is easier, when the user experience would be better with a server side integration."
+msgstr ""
+
+msgid "Only do things if they can be made fast."
+msgstr ""
+
+msgid "Don’t introduce components or features to AMP that can’t reliably run at 60fps or hinder the instant load experience on today’s most common mobile devices."
+msgstr ""
+
+msgid "Prioritise things that improve the user experience – but compromise when needed."
+msgstr ""
+
+msgid "Some things can be made fast and are still a terrible user experience. AMPs should deliver a fantastic user experience and speed is just one part of that. Only compromise when lack of support for something would stop AMP from being widely used and deployed."
+msgstr ""
+
+msgid "No whitelists."
+msgstr ""
+
+msgid "We won’t give any special treatment to specific sites, domains or origins except where needed for security or performance reasons."
+msgstr ""
+
+msgid "Discover more..."
+msgstr ""
+
+msgid "Create your first AMP page"
+msgstr ""
+
+msgid "Not sure how to get started? In this tutorial, you’ll learn how to create a basic AMP HTML page, how to stage it and validate that it’s AMP compliant, and finally ..."
+msgstr ""
+
+msgid "In this tutorial, you'll learn how to use data binding and expressions to build awesome, interactive AMP pages with amp-bind ..."
+msgstr ""
+
+msgid "Styling and layout on AMP HTML pages is very similar to normal HTML pages – in both cases, you'll use CSS."
+msgstr ""
+
+msgid "Get Started with Tutorials"
+msgstr ""
+
+msgid "AMP Websites"
+msgstr ""
+
+msgid "AMP Email"
+msgstr ""
+
+msgid "advertisers"
+msgstr ""
+
+msgid "publishers"
+msgstr ""
+
+msgid "e-commerce"
+msgstr ""
+
+msgid "nonprofit"
+msgstr ""
+
+msgid "social-media"
+msgstr ""
+
+msgid "Magazine article"
+msgstr ""
+
+msgid "Breaking the status quo"
+msgstr ""
+
+msgid "This example of magazine article demonstrates how to build an immersive reading experience by adding animations that enhance the written content on the page."
+msgstr ""
+
+msgid "<code>amp-position-observer</code> in combination with <code>amp-animation</code> allows a smooth \"scrolly-telling\" experience with minimal effort and without any custom JavaScript."
+msgstr ""
+
+msgid "Experience Demo"
+msgstr ""
+
+msgid "Band page"
+msgstr ""
+
+msgid "Olga and the kings"
```

Comment:
we probably should leave this untranslated as long as we don't translate the actual sample